core/src/main/scala/com/mesosphere/usi/core/logic/MesosEventsLogic.scala

```diff
@@ -108,9 +122,10 @@ private[core] class MesosEventsLogic(mesosCallFactory: MesosCalls) extends Impli
         val (schedulerEventsBuilder, _) =
           offersList.asScala.foldLeft((SchedulerEventsBuilder.empty, pendingLaunch)) {
             case ((builder, pending), offer) =>
-              val (matchedPodIds, offerMatchSchedulerEvents) = matchOffer(offer, pendingLaunch.flatMap { podId =>
-                specs.podSpecs.get(podId)
-              }(collection.breakOut))
+              val (matchedPodIds, offerMatchSchedulerEvents) = matchOffer(
```

Comment:
why did we drop the `(collection.breakOut)` ?

Reply:
using the breakout we would get a `Seq[PodSpec]` instead of `Set[PodSpec]`. is there any advantage in doing that since we can already pass the `Set` as-is ?